### PR TITLE
Fixing Css in Create Assignments + Custom Lessons Page

### LINKED ISSUE #80: Fix suspicious CSS in Create Assignments
-the calender
-student cards kinda jank
-also the word groups arent centered lol
-also someone slapped in another words component so love that for me

### PR DESCRIPTION
**Pull request by AngelaLuo49021**
## Description :star2:

some general css fixes like the word groups are no longer off centered and the font size for phonics is now fixed
Also the pop up for word group and the pop up for custom lessons isn't stuck in one place now either. 
Fixes #80
- Create Assignments page is tablet compatible now!
- also in exchange for kool tablet mode, everything on web mode is slightly larger than before and anisha approved it so its all chill
- General styling so its closer to Anisha's designs
- Originally, I made it so you can shrink the screen as much as you want and its still pretty good but when i did that, ipad mode was like a bit smaller than I would like. So tradeoff is that a screen minimized past half size is not great but ipad mode is hecka thriven
- tbh not many teachers will be assinging lessons on their phone so it shouldn't be an issue
Screen Shot 2020-05-09 at 1 32 23 PM
Screen Shot 2020-05-09 at 1 32 48 PM
Screen Shot 2020-05-09 at 1 33 15 PM
Screen Shot 2020-05-09 at 1 36 38 PM

[Link to tablet video](https://drive.google.com/file/d/1QNc-j8tszhWwIuNt8ChO0UpOGv1R8LUW/view?usp=sharing)
Also i was chewing my gum really loudly in that video and it sounds disgusting so just turn ur volume off lmao

### CODE REVIEW

File: src/components/DatePicker/DatePicker.scss [hunk 1]
```diff
@@ -30,36 +31,41 @@
...
   font-weight: bold;
-  font-size: 1em;
+  font-size: 1vw;
   color: black;
 }
 .react-calendar__month-view__weekdays__weekday {
   padding: 0.5em;
+  background-color:$assignments-font-color;
+  color: white;
+  font-size: 1rem;
```
Comment (chyku) [inline src/components/DatePicker/DatePicker.scss:68]: what an interesting font size choice bc doesnt this just make it the same size as the parent

File: src/components/InvalidAssignment/InvalidAssignment.scss [hunk 2]
```diff
@@ -1,45 +1,46 @@
...
   transition-duration: 0.3s;
   font-weight: bold;
   margin: auto;
 }
 
-.CloseMessage:hover {
+.close-message:hover {
   background-color: $close-button-color;
   color: white;
 }
```
Comment (chyku) [inline src/components/InvalidAssignment/InvalidAssignment.scss]: you could probably nest this inside `.close-message` if you want

File: src/components/LessonCardsDisplay/LessonCardsDisplay.scss [hunk 3]
```diff
@@ -1,32 +1,48 @@
...
+  max-height: 100%;
+  max-width: 100%;
+  width: auto;
+  height: auto;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
```
Comment (chyku) [inline src/components/LessonCardsDisplay/LessonCardsDisplay.scss:44]: what are these all here for, what are they needed for bc they seem like the kinds of things ppl add to make something not janky

File: src/components/StudentSelector/StudentSelector.js [hunk 4]
```diff
@@ -74,11 +74,11 @@ function StudentSelector(props) {
     newChooseAll[index] = !newChooseAll[index];
     setChooseAll(newChooseAll);
   }
-
+  //the key in the DropdownButton is hacky fix to the issue that the dropdown menu doesnt close upon selection
   return (
     <div className="student-selector">
       <DropdownButton
-        id="deployment-ddown"
+        id="depoyment-ddown"
```
Comment (chyku) [inline src/components/StudentSelector/StudentSelector.js]: stop spelling this wrong

File: src/components/StudentSelector/StudentSelector.js [hunk 5]
```diff
@@ -74,11 +74,11 @@ function StudentSelector(props) {
     newChooseAll[index] = !newChooseAll[index];
     setChooseAll(newChooseAll);
   }
-
+  //the key in the DropdownButton is hacky fix to the issue that the dropdown menu doesnt close upon selection
```
Comment (chyku) [inline src/components/StudentSelector/StudentSelector.js]: is this still a thing? i thought this was fixed

File: src/components/StudentSelector/StudentSelector.scss [hunk 6]
```diff
@@ -1,64 +1,73 @@
...
-    border-radius: 1em;
+    border-radius: 1vw;
 }
 
-#deployment-ddown {
-    height: 2em;
-    width:32vw;
-    border-top-left-radius: 1em;
-    border-top-right-radius: 1em;
+#depoyment-ddown {
```
Comment (chyku) [inline src/components/StudentSelector/StudentSelector.scss]: smh

File: src/components/WordGroupSelector/WordGroupSelector.js [hunk 7]
```diff
@@ -133,19 +134,22 @@ function WordGroupSelector(props) {
...
-            />
+          <div className="word-groups-margins">
+            <div onClick={() => handleClick(wordGroups[key][0], key, index)}>
+              <WordGroupIcon
+                name={key}
+                image={wordGroups[key][1]}
+                colored={cardColored[index]}
+                onClick={() => handleClick(wordGroups[key][0], key, index)}
+              />
+            </div>
```
Comment (chyku) [inline src/components/WordGroupSelector/WordGroupSelector.js:148]: was this your fix to being able to click on the outside of it?

File: src/pages/CreateAssignment/CreateAssignment.js [hunk 8]
```diff
@@ -16,6 +16,12 @@ import PhonicSelector from "../../components/PhonicSelector/PhonicSelector";
 import InvalidAssignment from "../../components/InvalidAssignment/InvalidAssignment";
 import LessonCardsDisplay from "../../components/LessonCardsDisplay/LessonCardsDisplay";
 
+/**
+ * CSS stuff i gotta do
+ *  resize lesson cards
+ * see if theres a way to keep font size and change the component size
+ * phonics cards be a little janky
+ */
```
Comment (chyku) [inline src/pages/CreateAssignment/CreateAssignment.js]: if u already did these pls remove

**Review by chyku — CHANGES_REQUESTED** [review]
depoyment

File: src/components/DatePicker/DatePicker.scss (hunk 1 above)
Comment (AngelaLuo49021) [reply src/components/DatePicker/DatePicker.scss:68]: same size as the root parent i think not the direct?

Comment (AngelaLuo49021) [reply src/components/DatePicker/DatePicker.scss:68]: if its not there, then the weekdays on an ipad are like hella tiny

File: src/components/LessonCardsDisplay/LessonCardsDisplay.scss (hunk 3 above)
Comment (AngelaLuo49021) [reply src/components/LessonCardsDisplay/LessonCardsDisplay.scss:44]: it veritcally and horixontally centers the plus icon. Super not janky method I took from an official css website 🥺

File: src/components/WordGroupSelector/WordGroupSelector.js (hunk 7 above)
Comment (AngelaLuo49021) [reply src/components/WordGroupSelector/WordGroupSelector.js:148]: yuh

File: src/pages/CreateAssignment/CreateAssignment.js (hunk 8 above)
Comment (AngelaLuo49021) [reply src/pages/CreateAssignment/CreateAssignment.js]: oops bad version control

**Review by chyku — APPROVED** [review]
good stuff

**Pull request merged**